src/browser/tab/element/mod.rs

```diff
@@ -352,6 +352,18 @@ impl<'a> Element<'a> {
         Ok(text)
     }
 
+    /// Get the full HTML contents of the element.
+    /// 
+    /// Equivalent to the following JS: ```element.outerHTML```.
+    pub fn get_content(&self) -> Result<String> {
+        let html = self.
+                call_js_fn("function() { return this.outerHTML }", vec![], false)?
```

Comment:
can you change this too to the puppeteer version?

Reply:
I don't think puppeteer has a built-in method to get the HTML of a selected element, not one that I found at least. 
On a StackOverflow [question](https://stackoverflow.com/questions/46431288/puppeteer-get-innerhtml) about this they used the evaluate method.

Reply:
ah i see thats ok then